INFO:root:Program started
INFO:root:Robot set up!
INFO:root:Odometry was started, PID: 25386:
INFO:root:Initial location: 0.6, 1.8, -1.5707963267948966
DEBUG:root:voy a girar
DEBUG:root:he girado, voy recto
DEBUG:root:he ido recto, voy a hacer un arco
DEBUG:root:LA R ES: 0.38704877208081223
DEBUG:root:24
WARNING:root:Traceback (most recent call last):
  File "/home/pi/oscar/main.py", line 34, in main
    mv.slalom_right(oscar, 0.4)
  File "/home/pi/oscar/actions/moves.py", line 172, in slalom_right
    arc(robot, [origin[0] - tile_size/2, origin[1]- 3*tile_size/2])
  File "/home/pi/oscar/actions/moves.py", line 57, in arc
    raise Exception('OH NOOO A WALL >:o')
Exception: OH NOOO A WALL >:o

WARNING:root:Abruptly stopped the robot!
INFO:root:Odometry was stopped
